.github/workflows/gulp.yml

```
@@ -0,0 +1,36 @@
+name: NodeJS with Gulp
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [12.x, 13.x, 14.x]
+    
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Use Node.js ${{ matrix.node-version }}
+      uses: actions/setup-node@v1
+      with:
+        node-version: ${{ matrix.node-version }}
+
+    - name: Build
+      run: |
+        npm install
+        gulp
```

If you can it would be clearer if these can be separate run commands.

Also, the readme says to use yarn to install, so I think the deps install should be `yarn`.

```suggestion
    - name: Install deps
      run: yarn

    - name: Build
      run: gulp
```

---

https://github.com/stellar/js-stellar-base/pull/457/commits/ae2b2d66a7a3d12c38ee76a7477e533a9d5ac44b